CONTRIBUTING.md

```diff
@@ -39,8 +39,7 @@ For more information about making well-organized pull requests, please read our
 * Include a test for any new module in the `modules/MODULENAME/test/`
   directory. You can look at other modules for examples of how to write tests.
 * Add your new automated tests to TravisCI in the `.travis.yml`. 
-* Make sure you run PHP codesniffer using the standards file in
-  `docs/LorisCS.xml` before sending any pull request, otherwise the automated tests will fail.
+* Make sure you run PHP codesniffer using the standards file in docs/LorisCS.xml by running `vendor/bin/phpcs --standard=docs/LorisCS.xml <path_to_changed_files>` before sending any pull request, otherwise the automated tests will fail.
```

Comment:
docs/LorisCS.xml should still have those quotes around it?

Reply:
I don't think so because the quotes in this edit refer to a bash command to run so it would sort of make it less clear.